core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1080,6 +1080,13 @@ public static long bitOr(long b0, long b1) {
     return b0 | b1;
   }
 
+  // ^
+
```

Comment:
We only have one implementation method for `^`, do we need this empty line ?

Reply:
You are right, Thank you @jinxing64, I have updated